src/definitions/elements/header.less

```
@@ -467,6 +467,11 @@
     .ui.bottom.attached.header {
         border-radius: 0 0 @attachedBorderRadius @attachedBorderRadius;
     }
+    & when (@variationHeaderSeamless) {
+        .ui.seamless.attached.header {
```

This would also remove the bottom border on a `seamless bottom attached header`. See https://jsfiddle.net/znLf2exm/

So please add `:not(.bottom)`
```suggestion
        .ui.seamless.attached:not(.bottom).header {
```

---

Done :+1: